tests/bench_xtimer_load/Makefile

```diff
@@ -0,0 +1,9 @@
+include ../Makefile.tests_common
+
+USEMODULE += xtimer
+
+# configure benchmark frequency
+TEST_HZ ?= 64LU
+CFLAGS += -DTEST_HZ=$(TEST_HZ)
```

Comment:
```Suggestion
CFLAGS += -DTEST_HZ=$(TEST_HZ)LU
```

Reply:
My compiler is otherwise complaining about trying to print an `int` where a `long int` is expected

Reply:
fixed, and changed the line above to just "64"